drivers/soundwire/slave.c

```diff
@@ -163,13 +163,13 @@ int sdw_acpi_find_slaves(struct sdw_bus *bus)
 
 			if (id.unique_id != id2.unique_id) {
 				dev_dbg(bus->dev,
-					"Valid unique IDs %x %x for Slave mfg %x part %d\n",
+					"Valid unique IDs 0x%x 0x%x for Slave part_id 0x%04x, mfg_id 0x%04x\n",
```

Comment:
The order of part_id and mfg_id is wrong. It should ``mfg_id 0x%04x,  part_id 0x%04x``

Reply:
this was intentional, there are other logs where we write the part_id first.
If we put mfg_id first, we've got to change the other logs.